sys/include/xtimer/tick_conversion.h

```diff
@@ -12,9 +13,48 @@
  * @{
  * @file
  * @brief   xtimer tick <-> seconds conversions for different values of XTIMER_HZ
+ *
  * @author  Joakim Nohlgård <joakim.nohlgard@eistec.se>
+ * @author  Josua Arndt <jarndt@ias.rwth-aachen.de>
  */
 
+/*
+ * As tick_conversion.h is included in multiple files ENABLE_DEBUG can not be used
+ * as it gets redefined "[-Werror]"
+ * To use  ENABLE_DEBUG_TICKS enable module fmt in the project makefile
+ * USEMODULE += fmt
+ * */
+#define ENABLE_DEBUG_TICKS    (0)
+
+#if ENABLE_DEBUG_TICKS
```

Comment:
Maybe also check for `&& defined(MODULE_FMT)` as you use this module here and it might not be included.